Recast/PodcastDetailViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
```

Comment:
This file will definitely conflict with jack, maybe name this something else?

Reply:
i'll actually delete it haha and re-implement the downloads view